psi4/driver/p4util/procutil.py

```diff
@@ -25,23 +25,19 @@
 #
 # @END LICENSE
 #
-
 """Module with utility functions used by several Python functions."""
-from __future__ import print_function
 import os
 import ast
 import sys
 import pickle
 import inspect
+import warnings
 import collections
 
 from .exceptions import *
```

Comment:
I know this isn't part of your PR, but could you remove this wildcard also if you make any more changes to this file, please?

Reply:
fixed all the exception wildcards in #1393

Reply:
Great, thanks!